cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -797,3 +805,136 @@ func installedPkgDetailFromRelease(r *release.Release, ref *corev1.InstalledPack
 		CustomDetail: customDetailHelm,
 	}, nil
 }
+
+func splitChartIdentifier(chartID string) (repoName, chartName string, err error) {
+	// getUnescapedChartID also ensures that there are two parts (ie. repo/chart-name only)
+	unescapedChartID, err := getUnescapedChartID(chartID)
+	if err != nil {
+		return "", "", err
+	}
+	chartIDParts := strings.Split(unescapedChartID, "/")
+	return chartIDParts[0], chartIDParts[1], nil
+}
+
+// CreateInstalledPackage creates an installed package.
+func (s *Server) CreateInstalledPackage(ctx context.Context, request *corev1.CreateInstalledPackageRequest) (*corev1.CreateInstalledPackageResponse, error) {
+	// Get the AppRepository for the available package.
+	// TODO: currently app repositories are only supported on the cluster on
+	// which Kubeapps is installed. #1982
+	chartID := request.GetAvailablePackageRef().GetIdentifier()
+	repoNamespace := request.GetAvailablePackageRef().GetContext().GetNamespace()
+	repoName, chartName, err := splitChartIdentifier(chartID)
+	chartVersion := request.GetPkgVersionReference().GetVersion()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Perhaps adding a console message to log this fn has been invoked? We are doing so in many fn, but not in every one. Should we remove them (or set other log level), or leave them? 
I personally like having a log entry for each operation, but see what you think.